scvi/module/_vae.py

```diff
@@ -360,8 +353,20 @@ def generative(
 
         px_r = torch.exp(px_r)
 
+        if self.gene_likelihood == "zinb":
```

Comment:
are these related changes?

Reply:
Yes! I feel that returning the likelihood distribution in `generative` simplifies `loss`, and can be used elsewhere (e.g., in `sample`), without having to reconstruct the distribution.
However, this may not be a good idea for all models, as some have custom likelihoods that are not coded as `torch.distributions.Distribution` (e.g., MultiVI).
Do you think I should remove this?

Reply:
ah I see, no I think it makes sense since its parameterized as such w/ the gene_likelihood arg.

Reply:
but aren't there more changes to be done in this file? E.g., can we remove `get_reconstruction_loss`?

Reply:
and if this is the case, should we do it globally? And finally, with our negative binomial, not each parameter is identifiable from the distribution object, namely, `px_scale`

Reply:
we can remove the `get_reconstruction_loss`. A solution could be to feed `px_scale` and `library` in addition to `px_rate` to some `Distribution` object, that could then be used later on.
This mostly make sense for models which rely on `Distribution` to compute the reconstruction loss though, and I am not sure what we should do for other models.

Reply:
Done.